components/post_view/reaction/reaction.jsx

```diff
@@ -207,28 +271,26 @@ export default class Reaction extends React.PureComponent {
                     />
                 );
             } else {
-                className += ' post-reaction--read-only';
+                //className += ' Reaction--read-only';
```

Comment:
There's some left over code that can be cleaned up.

Reply:
Thanks